src/app/test/api/http/unit/handlers/v1/garden_test.py

```diff
@@ -360,6 +382,8 @@ def test_auth_enabled_returns_permitted_gardens(
         assert len(response_body) == 1
         assert response_body[0]["id"] == str(garden_permitted.id)
         assert "connection_params" in response_body[0].keys()
+        assert len(response_body[0]["systems"]) == 1
+        assert response_body[0]["systems"][0]["name"] == system.name
```

Comment:
These checks are getting a little unwieldy, but may as well also check the namespaces, since that was the other data point that was missing before this change.